src/Microsoft.Identity.Client/Platforms/net45/NetDesktopWebUIFactory.cs

```diff
@@ -16,6 +17,11 @@ public IWebUI CreateAuthenticationDialog(CoreUIParent parent, RequestContext req
                 return new SilentWebUI(parent, requestContext);
             }
```

Comment:
Do we have any e2e tests for this?

Reply:
Yes, I have added unit tests for the factory here: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/c4336ad98cb2ecbba574dac65f5fffad473495d2/tests/Microsoft.Identity.Test.Unit.net45/WebUITests/NetDesktopWebUIFactoryTests.cs

Reply:
(there is an equivalent one for the NetCoreWebUiFactory)